### ISSUE TITLE
Gooey didn't work

### ISSUE THREAD

**Issue by miek770** [Nuitka/Nuitka#806]
Hi,

Has anyone had any success compiling and running a Gooey app? My build succeeds and the configuration window opens, but executing the program (once the parameters have been chosen in Gooey) produce the following error:

```
  File "C:\Users\michel\gits\nuitka-test\hello.dist\hello.exe", line 1
SyntaxError: Non-UTF-8 code starting with '\x90' in file C:\Users\michel\gits\nuitka-test\hello.dist\hello.exe on line 1, but no encoding declared; see http://python.org/dev/peps/pep-0263/ for details
```

This is very similar to #478.

I'm using the development version of Nuitka (installed from source - `git clone ...` followed by `python setup.py install` in a `venv`):

```
> python -m nuitka --version
0.6.9rc6
Python: 3.8.5 (tags/v3.8.5:580fbb0, Jul 20 2020, 15:57:54) [MSC v.1924 64 bit (AMD64)]
Executable: C:\Users\michel\gits\nuitka-test\Scripts\python.exe
OS: Windows
Arch: x86_64
```

To reproduce, use this slighly modified version of the *hello world" example (with `Gooey` installed of course, version 1.0.4 in my case):

```
from gooey import Gooey, GooeyParser


@Gooey
def main():
    parser = GooeyParser()
    parser.add_argument("Filename", widget="FileChooser")
    args = parser.parse_args()
    print(args.Filename)


if __name__ == "__main__":
    main()
```

Here is my build script (a `make.py` file) - it's for a slightly more complex than the *hello world* example):

```
from subprocess import run
from pathlib import Path
from shutil import rmtree, copy, copytree
from sys import platform
from os import environ, makedirs
from shutil import make_archive


# Either import or define the application name
from main import __title__
# __title__ = ""

# Either import or define the version string
from main import __version__
# __version = ""

main_file = "main.py"

gooey_path = Path("..", "Lib", "site-packages", "gooey")

data = {
    "codices",
    "configs",
}

options = [
    "--standalone",  # No Python installation required; implies --recurse-all
    "--plugin-enable=numpy",  # Needed for Gooey
    "--mingw64",  # Needed on Windows
    # "--windows-disable-console",  # On Windows, disable the console (best with GUI)
    # "--show-progress",
    "--assume-yes-for-downloads",
    "--remove-output",  # Delete the build folder
    "--verbose",
]

arch = environ["PROCESSOR_ARCHITECTURE"]
nuitka_dist_path = Path(f"{main_file.replace('.py', '')}.dist")
final_dist_path = f"{__title__}_{platform}_{arch}_{__version__}"


def main():
    # Remove the previous build
    try:
        rmtree(final_dist_path)
    except FileNotFoundError:
        pass

    # Build using Nuitka
    run(["python", "-m", "nuitka",  *options, main_file]).check_returncode()

    # Rename the main executable to the application title
    Path(nuitka_dist_path, main_file.replace('.py', '.exe')).rename(
        Path(nuitka_dist_path, f"{__title__}.exe"))

    # Rename the output folder to the full build title
    Path(nuitka_dist_path).rename(final_dist_path)

    # Copy the data folders
    for folder in data:
        copytree(Path(folder), Path(final_dist_path, folder))

    # Create Gooey's stuff
    makedirs(Path(f"{final_dist_path}", "gooey"))
    copy(
        gooey_path,
        Path(f"{final_dist_path}", "gooey"),
    )

    # Create ZIP archive
    make_archive(f"{final_dist_path}.zip", "zip", final_dist_path)

if __name__ == "__main__":
    main()
```

Thanks.

Edit: If they really are incompatible for some reason, would there be an alternative to Gooey that works well with Nuitka (i.e.: an auto UI from the CLI script arguments and options)?

**Comment by JoshKoh** [comment]
Same issue here with Gooey. Using Nuitka 0.6.8.4 64-bits. Compiled into .exe but on execution after parameters are selected, generates the SyntaxError: Non-UTF-8 code starting with '\x90'. Not sure if Gooey is using some non-standard UTF-8 in the code or Nuitka is picking up the wrong stuff? Would be nice if someone have a crack at figuring this out.

**Comment by kayhayen** [comment]
@JoshKoh use 0.6.9 or I won't care.

**Comment by kayhayen** [comment]
I didn't reproduce the issue, all I had on Python3.8 with Windows were:

Repeated reports of different DLLs for MS runtime. Seems there are 3+ different ones used, not too healthy, but Nuitka picks the latest one. For some files, e.g. MS runtimes, I guess, it may not even be worth to make it a warning, as it's going to work out that way.

BTW: I ignored the numpy and qt-plugin warnings, and got away with it. Normally on Windows that qt plugin for platforms is required, not sure why it worked without. Same for numpy, but that's not used apparently, so that was allowed. Not saying this is good, PyQt will not be able to load image formats, but that shouldn't cause any issue.

Also data files were needed, "languages" and "images" from gooey are needed in the dist folder, so these are now copied with the upcoming versions.

I think the changes for `sys.executable` might have fixed it. Some programs need it to point to the dist folder, it should be None, and Nuitka as of recently, points it to there, but not to the actual binary. Maybe that disables attempts at re-executing with Python, or something like that.

Maybe now that I think of it, looking at your stack, maybe you were running the exe yourself with Python, which would explain it too.... but anyway, it's gone it seems.

**Comment by kayhayen** [comment]
This is on factory and will be in the next release http://nuitka.net/doc/factory.html

**Comment by kayhayen** [comment]
Due to the smallness, I included it in 0.6.9.6 hotfix.
